pkgs/by-name/jo/joomla/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  fetchurl,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "joomla";
+  version = "5.2.3";
+
+  src = fetchurl {
+    url = "https://downloads.joomla.org/cms/joomla${lib.versions.major finalAttrs.version}/${
+      lib.replaceStrings [ "." ] [ "-" ] finalAttrs.version
+    }/Joomla_${
+      lib.replaceStrings [ "." ] [ "-" ] finalAttrs.version
+    }-Stable-Full_Package.tar.gz?format=gz";
+    sha256 = "sha256-j8qToPPxqLuxRkvCdqrVJNDO341PVDex/Vh9+ZV5HAA=";
+  };
+
+  sourceRoot = ".";
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir $out
+    cp -r * $out/
+  '';
```

Comment:
runHook {pre,post}Install

Reply:
:sweat_smile: true! Added in the latest commit! Thanks!